### PR TITLE
[Update]: Add Salesforce as a provider

### PR DESCRIPTION
**Pull request by mikah13**

### CODE REVIEW

**Comment by pilcrowonpaper** [comment]
Can you share the link to the Salesforce docs?

**Comment by mikah13** [comment]
here you go [doc](https://help.salesforce.com/s/articleView?id=sf.remoteaccess_oauth_web_server_flow.htm&type=5)

I also want to discuss if we should put `signature`, `id_token`, and `id` inside the ResponseToken

**Comment by pilcrowonpaper** [comment]
Bad timing 😅 

I've updated the library in few ways:

- `scope` option is now inside `createAuthorizationURL()`
- I've removed `getUser()` methods
- I've added documentation for each provider

**Comment by mikah13** [comment]
No problem. This looks great and I will update accordingly to the new changes.

**Comment by mikah13** [comment]
@pilcrowOnPaper can you please review this again? Thank you

**Comment by pilcrowonpaper** [comment]
Can you update the docs?

**Comment by mikah13** [comment]
Sure I will

**Comment by pilcrowonpaper** [comment]
Can you also update the `sidebar` field in `docs/malta.config.json` to include Salesforce?

**Comment by mikah13** [comment]
Yes done

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Arctic
+****# Arctic
```
Comment (pilcrowonpaper) [inline README.md]:
```suggestion
# Arctic
```

File: docs/pages/providers/salesforce.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Salesforce"
+---
+
+# Salesforce
+
+Implements OpenID Connect.
+
+For usage, see [OAuth 2.0 provider with PKCE](../oauth2-pkce.md).
```
Comment (pilcrowonpaper) [inline docs/pages/providers/salesforce.md]:
```suggestion
For usage, see [OAuth 2.0 provider with PKCE](/guides/oauth2-pkce).
```

**Review by pilcrowonpaper — APPROVED** [review]

**Comment by pilcrowonpaper** [comment]
Thanks!

**Pull request merged**